include/wpe/renderer-backend-egl.h

```diff
@@ -101,6 +101,10 @@ WPE_EXPORT
 EGLNativeDisplayType
 wpe_renderer_backend_egl_get_native_display(struct wpe_renderer_backend_egl*);
 
+WPE_EXPORT
+uint32_t
+wpe_renderer_backend_egl_get_display_platform(struct wpe_renderer_backend_egl*);
```

Comment:
I would remove the display from the name just get_platform or even get_native_platform for consistency with get_native_display.

Reply:
`eglGetPlatformDisplay` parameters are named `platform`, and `native_display`, so `get_platform` would make sense, but I wanted to make more obvious that it would be used to create the display.

Reply:
Right, and the documentation says:

platform
Specifies the native platform to obtain a display for.

native_display
Specifies the native display to connect to.

I think it's better to be consistent with other libwpe functions rather than egl API.

Reply:
Anyway, it was just a suggestion, I'm not opposed to get_display_platform

Reply:
OK, I changed to `get_platform`.